codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttEncoder.java

```diff
@@ -453,6 +453,7 @@ private static ByteBuf encodePublishMessage(
             return buf;
         } finally {
             propertiesBuf.release();
+            payload.release();
```

Comment:
I will need to check in details but if this is needed here it was also needed in the original code

Reply:
Yeah, there is a back comparability issue. Maybe you have some ideas to avoid it?